js/package.json

```diff
@@ -17,8 +17,8 @@
     "walletlink",
     "web3"
   ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "build/npm/dist/index.js",
```

Comment:
I recall that this change wasn't necessary in the past - wondering why it's here

Reply:
It's also not in the original pr

Reply:
We don't build to the `dist` folder anymore. Needed to make this change in order to successfully test local test dapp against these changes.